package.json

```diff
@@ -97,6 +97,7 @@
     "recursive-readdir-sync": "^1.0.6",
     "relay-compiler": "https://github.com/alloy/relay/releases/download/v1.3.0-artsy/relay-compiler-1.3.0-artsy.1.tgz",
     "relay2ts": "^0.2.0",
+    "snapshot-diff": "^0.2.1",
```

Comment:
This is a snapshot matcher that compares the diff between two objects, so if you add a single attribute to an object ( you'll see some of those tests below) the snapshot is much more focused on the real change, vs the full object which is more meaningless